### ISSUE TITLE
Package is using API from Combine with iOS 15 but Package.swift requires iOS 13 

### ISSUE THREAD

**Issue by mihaicris-adoreme** [Kolos65/Mockable#40]
Minimum deployment version iOS should be iOS 15 because package is using Combine API with minimum iOS 15. 
`$_invocations.receive(on: queue).values`

```swift
    public func verify(_ member: Member,
                       count: Count,
                       assertion: @escaping MockableAssertion,
                       timeout: TimeoutDuration,
                       file: StaticString = #file,
                       line: UInt = #line) async {
        do {
            let invocationsSequence = $_invocations.receive(on: queue).values
            try await withTimeout(after: timeout.duration) {
                for await invocations in invocationsSequence {
                    let matches = invocations.filter(member.match)
                    if count.satisfies(matches.count) {
                        break
                    } else {
                        continue
                    }
                }
            }
        } catch {
            let matches = invocations.filter(member.match)
            let message = """
            Expected \(count) invocation(s) of \(member.name) before \(timeout.duration)s, but was: \(matches.count)
            """
            assertion(count.satisfies(matches.count), message, file, line)
        }
    }
```

**Comment by hainayanda** [comment]
@Kolos65 I think if you want to support iOS 13.0 you will need to use a solution similar to what I did originally.

**Comment by Kolos65** [comment]
Already put up a fix in #42

The only problem was the `values` prop which could be easily implemented with an AsyncStream.

**Comment by Kolos65** [comment]
Fixed in **0.0.7**
